### PR TITLE
Fix #15 by adding . (dot) between event type name and version if missing

### LINKED ISSUE #15: Some events in CDEventConstants print incorrectly.
Some of the events in CDEventConstants have a small error where it does not have a . after the event name. For example, Testsuite Finished will print as "dev.cdevents.testsuite.finished0.1.0" rather than "dev.cdevents.testsuite.finished.0.1.0"

### PR DESCRIPTION
**Pull request by zaza**

### CODE REVIEW

File: src/test/java/dev/cdevents/constants/CDEventTypesTest.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+import org.junit.jupiter.api.Test;
+
+import dev.cdevents.constants.CDEventConstants.CDEventTypes;
+
+public class CDEventTypesTest {
+	@Test
+	void allEventTypeNamesMatchExpectedPattern() {
+		assertThat(CDEventTypes.values()).extracting(CDEventTypes::getEventType)
+				.allMatch(type -> type.matches("[a-z\\.]+\\.\\d+\\.\\d+\\.\\d+"));
```
Comment (afrittoli) [inline src/test/java/dev/cdevents/constants/CDEventTypesTest.java:13]: NIT: This could even be `dev\\.cdevents\\.[a-z]+\\.[a-z]+\\.\\d+\\.\\d+\\.\\d+"`

Comment (zaza) [reply src/test/java/dev/cdevents/constants/CDEventTypesTest.java]: Done.

**Comment by zaza** [comment]
@afrittoli seems like the `/approve` trick didn't work, this PR still requires a review.

**Comment by afrittoli** [comment]
@zaza heh, we don't have prow installed, silly me - if you rebase the PR I will merge it then

**Comment by zaza** [comment]
I'm sorry, I thought you're able to squash and merge the PR.

**Review by afrittoli — APPROVED** [review]

**Comment by rjalander** [comment]
@zaza @afrittoli, Currently refactoring the code the way we need to create a CDEvent as per the latest spec.
and here we used CDEvents Spec Version instead of event version as per the comment https://github.com/cdevents/sdk-java/pull/35#discussion_r1176412535

As per my knowledge will not be using  https://github.com/cdevents/sdk-java/blob/main/src/main/java/dev/cdevents/CDEventTypes.java, once this PR is available with all other events https://github.com/cdevents/sdk-java/pull/35

**Comment by zaza** [comment]
Closing the PR as requested.

**Pull request closed without merging**